index.html

```diff
@@ -70,5 +70,314 @@ <h1 class="headline_title">
         </ul>
       </div>
     </section>
-  </body>
-</html>
+    <section>
+        <div>
```

Comment:
- [ ] Kindly implement the [required changes](https://github.com/JansueT/Portfolio/pull/2/files#r1038804271) from the previous reviewer. 

![Screenshot from 2022-12-04 01-08-13](https://user-images.githubusercontent.com/60083437/205457741-8146cfa0-5086-4eaf-bb3f-5425bd1de97d.png)

Reply:
- [ OPTIONAL] PLease follow the previous review requirement :wink: